packages/apolloKeyv/src/index.ts

```diff
@@ -0,0 +1,39 @@
+import type {
+  KeyValueCache,
+  KeyValueCacheSetOptions,
+} from "@apollo/utils.keyvaluecache";
+import Keyv from "keyv";
+
+export class ApolloKeyv<V = string> implements KeyValueCache<V> {
+  private readonly keyv: Keyv<V>;
+
+  constructor(keyv?: Keyv<V>) {
+    this.keyv = keyv ?? new Keyv<V>();
+  }
+
+  async get(key: string) {
+    return this.keyv.get(key);
+  }
+
+  async set(key: string, value: V, opts?: KeyValueCacheSetOptions) {
+    // Maybe an unnecessary precaution, just being careful with 0 here. Keyv
+    // currently handles 0 as `undefined`. Also `NaN` is typeof `number`
+    if (typeof opts?.ttl === "number" && !Number.isNaN(opts.ttl)) {
+      this.keyv.set(key, value, opts.ttl * 1000);
```

Comment:
huh, i totally forgot until now that we had to multiply by 1000.  (do any tests we've written help confirm that this is accurate? docs do.)

Reply:
Yeah, the TTL tests indirectly show this
https://github.com/trevor-scheer/apollo-utils/blob/d05a548f5db3b3808d2dad4261e4a322eaffafae/packages/keyValueCache/src/__tests__/InMemoryLRUCache.test.ts#L21-L34